main/common/resources/full_database_creation_with_deletions.sql

```diff
@@ -0,0 +1,325 @@
+    id integer NOT NULL,
+    name character varying(20) NOT NULL,
+    enabled boolean
+);
+
+
+--
+-- Name: strategy_id_seq; Type: SEQUENCE; Schema: public;
+--
+
+CREATE SEQUENCE public.strategy_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+--
+-- Name: strategy_id_seq; Type: SEQUENCE OWNED BY; Schema: public;
+--
+
+ALTER SEQUENCE public.strategy_id_seq OWNED BY public.strategy.id;
+
+
+--
+-- Name: trade; Type: TABLE; Schema: public;
+--
+
+CREATE TABLE public.trade (
+    id integer NOT NULL,
+    open boolean NOT NULL,
+    open_time timestamp with time zone NOT NULL,
+    close_time timestamp with time zone,
+    side character varying(5) NOT NULL,
+    pair character varying(20) NOT NULL,
+    profit real,
+    spread real,
+    roi real,
+    strategy_id integer
+);
+
+
+--
+-- Name: trade_id_seq; Type: SEQUENCE; Schema: public;
+--
+
+CREATE SEQUENCE public.trade_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+--
+-- Name: trade_id_seq; Type: SEQUENCE OWNED BY; Schema: public;
+--
+
+ALTER SEQUENCE public.trade_id_seq OWNED BY public.trade.id;
+
+
+--
+-- Name: strategy id; Type: DEFAULT; Schema: public;
+--
+
+ALTER TABLE ONLY public.strategy ALTER COLUMN id SET DEFAULT nextval('public.strategy_id_seq'::regclass);
+
+
+--
+-- Name: trade id; Type: DEFAULT; Schema: public;
+--
+
+ALTER TABLE ONLY public.trade ALTER COLUMN id SET DEFAULT nextval('public.trade_id_seq'::regclass);
+
+
+--
+-- Name: classes_map classes_map_pkey; Type: CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.classes_map
+    ADD CONSTRAINT classes_map_pkey PRIMARY KEY (class_name);
+
+
+--
+-- Name: order_control order_pkey; Type: CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.order_control
+    ADD CONSTRAINT order_pkey PRIMARY KEY (order_id, pair);
+
+
+--
+-- Name: pair pair_pkey; Type: CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.pair
+    ADD CONSTRAINT pair_pkey PRIMARY KEY (pair_code);
+
+
+--
+-- Name: strategy strategy_pkey; Type: CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.strategy
+    ADD CONSTRAINT strategy_pkey PRIMARY KEY (id);
+
+
+--
+-- Name: trade trade_pkey; Type: CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.trade
+    ADD CONSTRAINT trade_pkey PRIMARY KEY (id);
+
+
+--
+-- Name: order_control trade_fkey; Type: FK CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.order_control
+    ADD CONSTRAINT trade_fkey FOREIGN KEY (trade_id) REFERENCES public.trade(id);
+
+
+--
+-- Name: trade trade_strategy_id_fkey; Type: FK CONSTRAINT; Schema: public;
+--
+
+ALTER TABLE ONLY public.trade
+    ADD CONSTRAINT trade_strategy_id_fkey FOREIGN KEY (strategy_id) REFERENCES public.strategy(id);
+
+
+--
+-- Data for Name: classes_map; Type: TABLE DATA; Schema: public;
+--
+
+COPY public.classes_map (class_name, class_code) FROM stdin;
```

Comment:
To confuso sobre essa tabela `classes_map`.
O detalhe das linhas está neste bloco e tbm no arquivo `class_map_rebuild.sql`. Qual deles é o correto?

Reply:
Posso assumir que os dois estão iguais?

Reply:
Precisa usar esse compando `copy`